game/src/ecs/entities/Entity.java

```diff
@@ -14,10 +15,13 @@ public class Entity {
     private static int nextId = 0;
     public final int id = nextId++;
     private HashMap<Class, Component> components;
+    private final Logger entityLogger = Logger.getLogger(this.getClass().getName());
```

Comment:
Hier ist es wieder mit Package.

Reply:
s.o. fixed in 52f4671d701138239da1e0e6cc93a3e66b8bd1a8